services/apps/alcs/src/alcs/application/application-document/application-document.entity.ts

```diff
@@ -43,6 +43,10 @@ export class ApplicationDocument extends BaseEntity {
   @Column({ nullable: true })
   documentUuid?: string | null;
 
+  @AutoMap(() => [String])
+  @Column({ default: [], array: true, type: 'text' })
+  visibilityFlags: string[];
```

Comment:
Another nitpick, to add a bit of type safety here it is possible to use Enum as type for ts